src/components/MainForm/Mainform.tsx

```diff
@@ -272,36 +262,33 @@ export default function Mainform() {
               </DialogContent>
             </Dialog>
             <TextField
+              error={!!errors.PhoneNumber}
               type="number"
               id="outlined-basic"
               label="Phone Number"
               variant="outlined"
               className={formClasses.phoneNumber}
               {...register("PhoneNumber", { required: true })}
+              helperText={
+                !!errors.PhoneNumber && <p>Phone Number Is Required!</p>
+              }
             />
-            {errors.PhoneNumber && (
-              <span className={formClasses.errorMsg}>
-                <b>Phone Number is required</b>
-                {(FormChecker = true)}
-              </span>
-            )}
+            {errors.PhoneNumber && (FormChecker = true)}
             <TextField
+              error={!!errors.email}
```

Comment:
I see you are using  FormChecker to disable and enable Button but luckily useForm provides us "isValid" we can use that .
 https://react-hook-form.com/api/useform/formstate

Reply:
done